src/main/java/at/hannibal2/skyhanni/features/garden/pests/PestAPI.kt

```diff
@@ -254,6 +255,7 @@ object PestAPI {
         if (pestDeathChatPattern.matches(event.message)) {
             lastPestKillTime = SimpleTimeMark.now()
             removeNearestPest()
+            PestKillEvent().post()
```

Comment:
```suggestion
            PestKillEvent.post()
```

To match with changing the event to an object